superset/charts/schemas.py

```diff
@@ -854,6 +854,7 @@ class ChartDataExtrasSchema(Schema):
     having_druid = fields.List(
         fields.Nested(ChartDataFilterSchema),
         description="HAVING filters to be added to legacy Druid datasource queries.",
+        deprecated= True,
```

Comment:
can we remove this field altogether?

Reply:
I worry that it might cause issues with older models? I can remove mentions of it in the code base though.

Reply:
We can probably remove this in the next major release. I think we just missed the 2.0 train. :)